frontend/packages/console-shared/src/components/formik-fields/DynamicFormField.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { FormProps } from 'react-jsonschema-form';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { AsyncComponent } from '@console/internal/components/utils';
+
+type DynamicFormFieldProps = FormProps<any> & {
+  name: string;
+  errors?: string[];
+  formDescription?: React.ReactNode;
+};
+
+const DynamicFormField: React.FC<DynamicFormFieldProps> = ({
+  name,
+  schema,
+  uiSchema,
+  errors,
+  formDescription,
+  formContext,
+}) => {
+  const [field] = useField(name);
+  const { setFieldValue } = useFormikContext<FormikValues>();
+
+  return (
+    <div className="row" style={{ marginTop: `16px` }}>
+      <div className="col-md-6 col-md-push-6 col-lg-6 col-lg-push-6">{formDescription}</div>
+      <div className="col-md-6 col-md-pull-6 col-lg-6 col-lg-pull-6">
+        <AsyncComponent
+          loader={() => import('../dynamic-form').then((c) => c.DynamicForm)}
```

Comment:
It brings in `react-json-schema-form` into the main vendor bundle which results in the main vendor bundle being `2.64MB` and then `analyze` fails because the allowed quota is `2.5MB`. That's why I had to make it Async.